maint/local_python/binding_c.py

```diff
@@ -2034,7 +2034,9 @@ def dump_validate_userbuffer_neighbor_vw(func, kind, buf, ct, dt, disp):
         dt += '[i]'
         dump_validate_datatype(func, dt)
     G.out.append("MPIR_ERRTEST_COUNT(%s, mpi_errno);" % ct)
-    G.out.append("MPIR_ERRTEST_USERBUFFER(%s, %s, %s, mpi_errno);" % (buf, ct, dt))
+    G.out.append("if (%s[i] == 0) {" % disp)
+    G.out.append("    MPIR_ERRTEST_USERBUFFER(%s, %s, %s, mpi_errno);" % (buf, ct, dt))
+    G.out.append("}")
```

Comment:
nit: I think these lines achieve the same output, but they are different in the 2 cases in this patch.
```
G.out.append("if (%s[i] == 0) {" % disp)
```
```
dump_if_open("%s[i] == 0" % disp)
```

Reply:
`dump_if_open(...)` will also insert a meta-line `INDENT`, and `dump_if_close()` will insert `DEDENT`, so the code in between are automatically indentation adjusted. When it is not optional and the line in between is short, direct print seems more readable.